### ISSUE TITLE
Some labels of filters not correct

### ISSUE THREAD

**Issue by peerlight** [pdir/maklermodul-bundle#14]
## Bug description
Some labels of filters dont have their correct "translation". 

We get "flaechen.anzahl.zimmer" instead of "Anzahl Zimmer", "flaechen.anzahl.ganze.zimmer", "geo.regionaler.zusatz". "Wohnfläche" and "Warmmiete" is ok.

No matter if indexed via backend or cron "flaechen.anzahl.zimmer" is in index-JSON

## Steps to reproduce
upgrade pdir/maklermodul-bundle 2.5.x -> 2.6.4?
upgrade pdir/maklermodul-sync-bundle 2.5.x -> 2.5.3?

There seems to be a transformation from key flaechen.anzahl_zimmer to flaechen.anzahl.zimmer. Has thsi something to do with the recent changes for "groups"?

## Expected behavior
Show correct labels, not transformed keys

## Screenshots
sorry, not able to upload. can send this via mail

## Environment
- PHP 7.3
- Contao 4.9.5 (language of site is "de", fallback active)
- pdir/maklermodul-bundle 2.6.4
- pdir/maklermodul-sync-bundle 2.5.3

**Comment by peerlight** [comment]
some additional info

Index of demo data files/maklermodul/data/00index-demo-00001.json has
```
      {"uriident":"klein-und-fein-sowie-bezahlbar-mit-platzhalterbild-14035",
        ...
        "flaechen-anzahl_zimmer":"1",
        ...
     }
  ],
  "filterConfig":{
     "groups":[
        {
           "key":"flaechen-anzahl_zimmer",
           "name":"Anzahl Zimmer"
        },
        ...
     ],
     "values":{
        "flaechen-anzahl_zimmer":[
           {
              "css":"1-2",
              "name":"1-2"
           },
           ...
```

But live data is indexed to 
```
    {"uriident":"neu-renovierte-2-raum-we-in-der-innenstadt-jetzt-sichern-1301-3-0089"
...
    "geo-regionaler_zusatz":"Zentrum"
    "flaechen-anzahl_zimmer":"2"
...}]
"filterConfig":
{"groups":[
    {"key":"flaechen-anzahl-zimmer"
    "name":"flaechen.anzahl.zimmer"}
...
    {"key":"geo-regionaler-zusatz"
    "name":"geo.regionaler.zusatz"}]
"values":
{"flaechen-anzahl-zimmer":[
    {"css":"2-3"
    "name":"2-3"}
    ...
```

mind the differences: _ - and .
 "flaechen-anzahl_zimmer":"2"
"key":"flaechen-anzahl-zimmer"
"name":"flaechen.anzahl.zimmer"

**Comment by peerlight** [comment]
Workaraound:

make use of a /contao/languages/de/makler_modul_mplus.php and change 

`$GLOBALS['TL_LANG']['makler_modul_mplus']['field_keys']['flaechen.anzahl_zimmer'] = 'Anzahl Zimmer';`

to 

`$GLOBALS['TL_LANG']['makler_modul_mplus']['field_keys']['flaechen.anzahl.zimmer'] = 'Anzahl Zimmer';`

**Comment by martin-netprofit** [comment]
Same problem here

**Comment by peerlight** [comment]
> Workaraound:
is not reliable

after indexing by cron 'flaechen.anzahl_zimmer' is there again ...

**Comment by peerlight** [comment]
Is this commit intended to fix this issue?
https://github.com/pdir/maklermodul-bundle/commit/2160b4403b3f5b80bd8dc7cc5762c10e6608ed2e

**Comment by peerlight** [comment]
Problem occurs when indexing is executed by cron with `$GLOBALS['TL_CRON']['hourly'][] = array('Pdir\MaklermodulSyncBundle\Module\Sync', 'estateIndex');` oder console `php vendor/bin/contao-console maklermodul:index`

I can "reset" to correct labels by indexing again from backend module "Maklermodul Setup"

So this could probably be fixed in MaklermodulSyncBundle - and should as people have payed for it

**Comment by seibtph** [comment]
We will check and solve it.

**Comment by peerlight** [comment]
Seems unresolved until today. Any news on this?

**Comment by peerlight** [comment]
Why are theses affected 
```
flaechen.anzahl_zimmer
geo.regionaler_zusatz
kontaktperson.tel_zentrale
zustand_angaben.baujahr
flaechen.anzahl_balkone
```
and these not?
```
flaechen.wohnflaeche
geo.plz
objektkategorie.objektart.wohnung.@wohnungtyp
```

**Comment by peerlight** [comment]
Problem persists with 

PHP 7.4
Contao 4.9.26
pdir/maklermodul-bundle 2.7.9
pdir/maklermodul-sync-bundle 2.6.3

**Comment by MDevster** [comment]
fixed in pdir/maklermodul-bundle 2.8.0 and pdir/maklermodul-sync-bundle 2.7.0
